cfme/tests/services/test_myservice.py

```diff
@@ -114,21 +88,20 @@ def test_crud_set_ownership_and_edit_tags(appliance, context, myservice):
         tags: service
     """
 
-    service_name, vm_name = myservice
+    service, _ = service_vm
     with appliance.context.use(context):
-        myservice = MyService(appliance, name=service_name, vm_name=vm_name)
-        myservice.set_ownership("Administrator", "EvmGroup-administrator")
-        myservice.add_tag()
-        with update(myservice):
-            myservice.description = "my edited description"
-        myservice.delete()
+        service.set_ownership("Administrator", "EvmGroup-administrator")
```

Comment:
TODO: 
1. more assertion that the tag is correctly assigned, that the ownership is correctly set, that the updated description is reflected in the UI, that the service is deleted.
2. parametrized on `ViaREST`